src/api/prop.js

```diff
@@ -42,3 +42,9 @@ export const string = create({
   deserialize: toNullOrString,
   serialize: toNullOrString
 });
+
+export const object = create({
+  default: {},
+  deserialize: JSON.parse,
```

Comment:
See `prop.array()` for what should probably be done here. They can share this function, too.